docs/changelogs/CHANGELOG-2.0.md

```diff
@@ -4,6 +4,8 @@
 
 ### Added
 
+- [#1332](https://github.com/epiphany-platform/epiphany/issues/1332) - Support for RHEL 8.5
```

Comment:
I think that these two lines should be joined.

Reply:
Joined.